edit.html

```diff
@@ -18,6 +18,7 @@
       <img class="img primary open" src="assets/logo-circle.svg" id=location-icon>
       <img class="img secondary" src="https://preview.exokit.org/[https://raw.githubusercontent.com/avaer/vrm-samples/master/vroid/male.vrm]/preview.jpg" id=profile-icon>
       <img class="img tertiary" src="https://preview.exokit.org/[https://raw.githubusercontent.com/avaer/vrm-samples/master/vroid/female.vrm]/preview.jpg" id=item-icon>
+      <img class="img tertiary" src="https://user-images.githubusercontent.com/29695350/101466101-5d219100-3906-11eb-9f7b-ccbf00492f3c.jpg" id=item-monetized-icon>
```

Comment:
Probably best to have this image in the repo.

Reply:
8f451f2